llvm/lib/CodeGen/TypePromotion.cpp

```diff
@@ -448,7 +448,7 @@ void IRPromoter::ExtendSources() {
       if (isa<Argument>(V))
         I->moveBefore(InsertPt);
       else
-        I->moveAfter(InsertPt);
+        I->moveAfter(&*InsertPt);
```

Comment:
Same as earlier - is this a code-transition-state bug, or is there some reason we've got to use the `Instruction *` overload here? (this one matters more)